### DISCUSSION TITLE
Local server instance threw Forbidden access error

### DISCUSSION THREAD

**Discussion by mfonnesbeck** [ravendb/ravendb#17114] [Q&A]
I was in the middle of development, using a local server instance which I had been using all morning, and suddenly while trying to connect and run a query from my code, I received the following error:

Forbidden access to earaya-emphoa-loc@https://a.emphoaloc.ravendb.community, CN=*.emphoaloc.ravendb.community does not have permission to access it or is unknown. Method: GET, Request: https://a.emphoaloc.ravendb.community/topology?name=earaya-emphoa-loc&first-topology-update
{"Type":"InvalidAuth","Message":"The supplied client certificate 'CN=*.emphoaloc.ravendb.community(Thumbprint: B75ECE4B367200712747C4B8E87619EF6FB2E5B7)' is unknown to the server. In order to register your certificate please contact your system administrator."}

My Certificates page in my server shows the following:
Server Certificate
57313052B7214AFBDD0C36C76A89B882FDE28D2F
SECURITY CLEARANCE : Cluster Node
VALID FROM : 2023-08-25
EXPIRATION : 2023-11-23
AUTO RENEWAL DATE : 2023-10-28
DATABASE PERMISSIONS : All

emphoaloc.client.certificate
E3C93F55772B1C9BC2A2E44FB7EEAD846BE44A26
SECURITY CLEARANCE : Cluster Admin
VALID FROM : 2023-06-08
EXPIRATION : 2028-06-15
DATABASE PERMISSIONS : All 

Did my certificate just renew?  If so, how can I download a new client certificate? Please help me understand what could have just happened.

**Comment by ayende** [comment]
Did you reset the server certificate?
It looks like you a trying to connect to the server using the _old_ server certificate ?

**Reply by mfonnesbeck** [reply]
As far as I know I did not. I don't even know how I would do it. I cannot see a place or a way to download the new certificate. How is this done with the local server? I see the manage certificates page and it shows the server and client certificate details, but no option to download one.

**Reply by ayende** [reply]
How are you authenticating the client ? Are you getting this from the API or the studio?

**Reply by mfonnesbeck** [reply]
When I started with the local server instance I was able to download a ZIP file with a certificate that I install on my Windows machine in order to connect to the studio through the browser and I use that certificate with the client API in code to connect with my application.

**Reply by ayende** [reply]
From where are you getting the error? From the browser or the API?

At any rate, try creating a new certificate and see if that works for you.

**Reply by mfonnesbeck** [reply]
I believe the API. I run my web application and it fails as it tries to run a query.
What is the process for creating a new certificate and downloading it?

**Reply by garayx** [reply]
you can check the following documentation pages

https://ravendb.net/docs/article-page/5.4/csharp/server/security/authentication/certificate-management#generate-client-certificate

https://ravendb.net/docs/article-page/5.4/csharp/studio/server/certificates/server-management-certificates-view#generate-client-certificate
